code-studio/src/js/levelbuilder_dsl.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file Main entry point for scripts used only in levelbuilder when editing
+ *       DSL-defined levels.
+ */
+/* global $ */
+'use strict';
+var initializeEmbeddedMarkdownEditor = require('./initializeEmbeddedMarkdownEditor');
+
+// Initialize markdown editors on page load
+$(function () {
```

Comment:
I prefer the long-from `$(document).ready(` which makes it a little clearer what's happening.